pkg/upgrade/uninstallation.go

```diff
@@ -27,6 +29,11 @@ const (
 // This includes DSCI, namespace created by operator (but not workbench or MR's), subscription and CSV.
 func OperatorUninstall(ctx context.Context, cli client.Client, platform cluster.Platform) error {
 	log := logf.FromContext(ctx)
+
+	if deleteError := removeDSC(ctx, cli); deleteError != nil {
+		return fmt.Errorf("error deleting DSC : %w", deleteError)
```

Comment:
since the `removeDSC` method already wrap the error, it is not necessary to wrap it again, just return it.